Sources/Networking/HTTPClient/HTTPClient.swift

```diff
@@ -269,6 +283,32 @@ private extension HTTPClient {
         }
     }
 
+    struct ExponentialRetryOptions: CustomStringConvertible {
+
+        let retriableStatusCodes: Set<HTTPStatusCode>
+        let baseBackoff: TimeInterval
+        let maxBackoff: TimeInterval
+        let maxNumberOfRetries: UInt
```

Comment:
I was wondering, do we need both a `maxBackoff` and `maxNumberOfRetries`? One or the other could be computed of the other + baseBackoff right?

Reply:
In my updates to some of the other PR comments, I've hardcoded the backoff intervals. After that change was done, most of this struct was redundant so it's been removed!